src/build-plugin.ts

```diff
@@ -6,14 +6,15 @@ import { maybeRewriteSourcemapWithReactProd, loadSourcemap } from "./index";
 
 export interface ReactSourcemapsPluginOptions {
   debug?: boolean;
+  preserve?: boolean;
 }
 
 const PLUGIN_NAME = "react-sourcemaps";
 // Iterates over the list of generated assets, skips any that are not sourcemaps
 // and attempts to rewrite them with React production sourcemaps.
 function rewireSourceMapsFromGeneratedAssetList(
   generatedAssets: string[],
-  options: ReactSourcemapsPluginOptions = { debug: false}
```

Comment:
the default should only be initialized in one place (the unplugin factory) so we avoid multiple initializers and possible subtle bugs